README.md

```diff
@@ -216,7 +221,16 @@ module "minio" {
                   }
 }
 ```
-
+### Key Management Secrets (KMS)
+The Key Management secrets engine provides a consistent workflow for distribution and lifecycle management of cryptographic keys in various key management service (KMS) providers.
```

Comment:
There are 2 other variables that should be mentioned in this context (`vault_address`, `vault_kms_approle_kv`).

Also, think that the example should show implementation not one of the variables.